sklearn/pipeline.py

```diff
@@ -868,6 +869,17 @@ def _validate_transformers(self):
                                 "transform. '%s' (type %s) doesn't" %
                                 (t, type(t)))
 
+    def _validate_transformer_weights(self):
+        if self.transformer_weights:
```

Comment:
Nit: I have a preference to early exiting, which reduces the levels of indentation:

```py
if not self.transformer_weights:
	return

transformer_names = ...
```

Reply:
Doesn't seem like a nit to me, I'm a big fan is fewer indentation levels.